birdhouse/README.rst

```diff
@@ -114,6 +114,37 @@ Disk: at least 100 TB, depending how much data is hosted on Thredds and Geoserve
 In general, the more users, the more cpu cores and memory needed.  The more data, more memory and bigger and faster disks needed.
 
 
+Local deployment for development
+--------------------------------
+
+To deploy the birdhouse stack for development, a self-signed SSL certificate is required:
+
+First create a self signed certificate as described in the `LetsEncrypt documentation <https://letsencrypt.org/docs/certificates-for-localhost/#making-and-trusting-your-own-certificates>`_ .
+Then add the certificate you just generated to the trust store as described `here <https://ubuntu.com/server/docs/security-trust-store>`_ .
+Finally, combine the generated key and certificate files together to use as the `SSL_CERTIFICATE` for this stack.
+
+The following should work to generate a self-signed ssl certificate on ubuntu 20.04+:
+
+.. code-block::
+
+    openssl req -x509 -out localhost.crt -keyout localhost.key \
+      -newkey rsa:2048 -nodes -sha256 \
+      -subj '/CN=localhost' -extensions EXT -config <( \
```

Comment:
Shouldn't this include a `-days N` as well to avoid it being expired after a few days?